modules/http-codec.lua

```diff
@@ -88,7 +88,7 @@ exports.encoder = function ()
   local encodeHead, encodeRaw, encodeChunked
 
   function encodeHead(item)
-    if not item then return end
+    if not item or item == "" then return end
```

Comment:
Also we should probably check the type of item and throw an error with a helpful error message if it's not a table (After this line where `nil` and `""` are valid inputs)